pkg/devserver/api.go

```diff
@@ -421,6 +423,12 @@ func (a devapi) OTLPTrace(w http.ResponseWriter, r *http.Request) {
 
 				}
 
+				if val, ok := sattr[consts.OtelSysFunctionHasAI]; ok {
```

Comment:
Is this idiomatic go? So far from my typescript inclinations to `!!sattr[consts.OtelSysFunctionHasAI]`!

Reply:
Yep! This is perfect